main_test.go

```diff
@@ -15,6 +15,16 @@ func TestBaseCommandsAreRegistered(t *testing.T) {
 	baseManager := cmd.BuildBaseManager("tranor", "", "", nil)
 	manager := buildManager("tranor")
 	for name, expectedCommand := range baseManager.Commands {
+		var skip bool
+		for _, c := range baseCommandsToRemove {
```

Comment:
manager.Register fails if we reuse a command name, so we need to remove target-set before being able to register it.

We also need the ``TestTargetSetIsRegistered`` to ensure that we overwrite the command that comes with tsuru (the command type). I renamed the test to make it more clear.